ui/test/unit/appointments/controllers/manage/list/appointmentsListViewController.spec.js

```diff
@@ -602,8 +603,8 @@ describe('AppointmentsListViewController', function () {
             scope.sortAppointmentsBy('additionalInfo');
             expect(scope.sortColumn).toEqual('additionalInformation');
             expect(scope.filteredAppointments.length).toEqual(2);
-            expect(scope.filteredAppointments[0].additionalInfo).toEqual(otherAppointment.additionalInfo);
-            expect(scope.filteredAppointments[1].additionalInfo).toEqual(appointment.additionalInfo);
+            expect(scope.filteredAppointments[0].additionalInfo).toEqual(appointment.additionalInfo);
+            expect(scope.filteredAppointments[1].additionalInfo).toEqual(otherAppointment.additionalInfo);
```

Comment:
@buvaneswari-arun please take care of the indentation here

Reply:
The build passed Bindu - so I assumed the indentations were correct, as there is a special check for indentation too. Not sure how to proceed on this.